gearcatalog/frontend/src/components/gear/Form.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import { addGear } from "../../actions/gear-list";
+import { Link } from "react-router-dom";
+import Input from '../common/Input';
+import { useInput } from '../../hooks/useInput';
+
+type FormProps = {
+  // TODO: find correct gear type
+  addGear: (gear) => void,
```

Comment:
Should you have a type specified for the callback's parameter? Or is the type name `gear`? If the latter, why not `Gear`?

Reply:
Yes, I should have the type of the parameter listed. I was having issues finding what the correct type for my `Gear` is. I was going to do more research and come back to it.